src/main/java/uk/gov/hmcts/darts/arm/service/impl/ArmApiServiceImpl.java

```diff
@@ -38,9 +35,29 @@ public UpdateMetadataResponse updateMetadata(String externalRecordId, OffsetDate
             .build();
 
         return armApiClient.updateMetadata(
-            String.format("Bearer %s", armTokenResponse.getAccessToken()),
+            getArmBearerToken(),
             armUpdateMetadataRequest
-        ).getBody();
+        );
+    }
+
+    @Override
+    public InputStream downloadArmData(String externalRecordId, String externalFileId) {
+        byte[] response = armApiClient.downloadArmData(
+            getArmBearerToken(),
+            armApiConfigurationProperties.getCabinetId(),
+            externalRecordId,
+            externalFileId
+        );
+        return new ByteArrayInputStream(response);
+    }
+
+    private String getArmBearerToken() {
+        ArmTokenResponse armTokenResponse = armTokenClient.getToken(new ArmTokenRequest(
+            armApiConfigurationProperties.getArmUsername(),
+            armApiConfigurationProperties.getArmPassword(),
+            "password"
```

Comment:
probably need a constant for this we also use it in AccessTokenClient

Reply:
I've added an enum `GrantType`.  Added it in both `main` and `functionalTest`